docs/README.md

```diff
@@ -158,6 +166,8 @@ minimum_versions:
 
 When you wish to manage annotations or labels for the namespaces you are installing into with Reckoner, this `namespace_management` block to define default namespace metadata and the whether or not it should overwrite the values that exist.
 
+`namespace_management` blocks can be defined at the top level or at the chart level. By default, the top level `default` metadata will be used for all namespaces and any `metadata.annotations` or `metadata.labels` set in the charts will be additive. However, if `settings.overwrite` is `True` then the `metadata` block from the chart will replace all overwrite any matching labels or annotations values.
```

Comment:
```suggestion
`namespace_management` blocks can be defined at the top level or at the chart level. By default, the top level `default` metadata will be used for all namespaces and any `metadata.annotations` or `metadata.labels` set in the charts will be additive. However, if `settings.overwrite` is `True` then the `metadata` block from the chart will replace any matching labels or annotations values.
```

Reply:
Also, I think we should document the fact that the chart-level namespace_management block cannot `remove` or `delete` course-level annotations/labels. It can only `modify` them.